packages/main/src/ButtonBadgeTemplate.tsx

```diff
@@ -0,0 +1,6 @@
+import type Button from "./Button.js";
+import Tag from "./Tag.js";
+
+export default function ButtonTemplate(this: Button) {
```

Comment:
Here `this` must be `ButtonBadge`  (but works with Button too, because they both have a `text` property). We can fix this later in another PR along with something else, do not push again just for this.

Reply:
https://github.com/SAP/ui5-webcomponents/pull/10705